src/onepasswordconnectsdk/client.py

```diff
@@ -44,25 +44,53 @@ def create_session(self):
 
     def build_headers(self):
         """Builds the headers needed to make a request to the server
-        Parameters:
 
         Returns:
-        dict: The 1Password Connect API request headers
+            dict: The 1Password Connect API request headers
         """
 
         headers = {}
         headers["Authorization"] = f"Bearer {self.token}"
         headers["Content-Type"] = "application/json"
         return headers
 
-    def get_item(self, item_id: str, vault_id: str):
+    def get_item(self, item: str, vault: str):
+        """Get a specifiv item
+        
+        Args:
+            item (str): the id or title of the item to be fetched
+            vault (str): the id or name of the vault in which to get the item from
+
+        Raises:
+            FailedToRetrieveItemException: Thrown when a HTTP error is returned
+            from the 1Password Connect API
+
+        Returns:
+            Item object: The found item 
+        """
+
+        vault_id = vault
+        if self._is_valid_UUID(vault) is False:
+            vault_id = self.get_vault_by_title(vault).id
```

Comment:
we should handle if no vault is found by title

Reply:
This is already handled by `get_vault_by_item` function. It throws an error, similar to the one thrown when no items with the given title are found. Check [this snippet](https://github.com/1Password/connect-sdk-python/blob/2799f30aeaf2e9226bc0d935417b471d0bda2aed/src/onepasswordconnectsdk/client.py#L107) for `get_vault_by_title`, compared to [this snippet](https://github.com/1Password/connect-sdk-python/blob/2799f30aeaf2e9226bc0d935417b471d0bda2aed/src/onepasswordconnectsdk/client.py#L272) for `get_item_by_title`

Reply:
Thanks!